ha/remote_execution/ssh_communication.py

```diff
@@ -0,0 +1,52 @@
+import paramiko
+
+from cortx.utils.ssh import SSHChannel
+from ha.remote_execution.remote_executor import RemoteExecutor
+from ha.const import RA_LOG_DIR
+from cortx.utils.log import Log
+
+class SSHRemoteExecutor(RemoteExecutor):
+    '''class which enables remote communication using SSH'''
+
+    def __init__(self, hostname, username, port=22, password=None):
+        '''init method'''
+        super(self, SSHRemoteExecutor).__init__(hostname, port)
+        self._username = username
+        self._password = password
+        try:
+            self._ssh_client = SSHChannel(self._hostname, self._port, \
+                                            self._username, self._password)
+        except Exception as e:
+            Log.error('SSHRemoteExecutor, some error occured while connecting \
+                        to SSH channel {e}')
+
+    def execute(self, command: str) -> int:
+        '''
+        Communicates to remote node by code execution
+
+        Parameters:
+            command: command to be executed on a remote node
+
+        Return:
+            int
+
+        Exceptions:
+                RemoteExecutorError
+        '''
+        ret_code = 0
+        res = None
+        Log.info(f'Execting command: {command} on a remote host {self._hostname}')
+        try:
+            ret_code, res = self._ssh_client.execute(command)
```

Comment:
It doesn't return error message. Ideally should return error message too which can be logged to debug the issue.

Reply:
The execute method of SSHChannel(cortx utils) will directly return the error message and will be captured in res. And now I have modified the code so that it will return error message now